main.go

```diff
@@ -27,10 +33,19 @@ type healthcheck struct {
 
 func main() {
 	cfg := config{}
+  env.Parse(&cfg)
+
 	hc := healthcheck{}
 
 	// default config for our own healthcheck
-	defcfg := config{Application: "healthcheck", Status: 200, Metrics: "{}"}
+	defcfg := config{Application: "healthcheck", Status: 200, Metrics: "{}" }
+
+	beeline.Init(beeline.Config{
```

Comment:
maybe consider breaking the config out into its own variable, to make it easier to parse?

```go
beeCfg := beeline.Config{
    WriteKey: cfg .HoneyKey,
    ...
}

beeline.Init(beeCfg)
```

Reply:
oh, also, you might consider checking that `cfg.HoneyKey` looks like a valid API key, and only `beeline.Init()` if it is?  i looked at the beeline docs and it’s not clear to me how they handle invalid configs